src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/OnceInitializedOnceDisposedUnderLockAsync.cs

```diff
@@ -120,7 +118,7 @@ private async Task<T> ExecuteUnderLockCoreAsync<T>(Func<CancellationToken, Task<
 
                 try
                 {
-                    T result = default;
+                    T result = default!;
```

Comment:
I assume we want to say that T may be null here?

Reply:
We can't unfortunately, as `T` is an unconstrained generic type.